openreview/venue/process/post_submission_process.py

```diff
@@ -3,22 +3,41 @@ def process(client, invitation):
     domain = client.get_group(invitation.domain)
     venue_id = domain.id
     submission_venue_id = domain.content['submission_venue_id']['value']
+    public = domain.content['public_submissions']['value']
+    venue_name = domain.content['title']['value']
 
     now = openreview.tools.datetime_millis(datetime.datetime.utcnow())
     cdate = invitation.cdate
 
     if cdate > now:
         ## invitation is in the future, do not process
         print('invitation is not yet active', cdate)
-        return    
+        return
 
     def post_submission_edit(submission):
 
+        updated_note = openreview.api.Note(
+            id=submission.id
+        )
+
+        if public and submission.odate is None:
```

Comment:
instead of using the `public` variable, can we check if the readers of the submission is everyone?